### PR TITLE
Renamed owned to held.

### PR DESCRIPTION
**Pull request by roger3cev**
Some bits of clean up in "contracts" module.

(cherry picked from commit 4e01a63)

Fixed up compilation errors.

### CODE REVIEW

File: contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt [hunk 1]
```diff
@@ -51,15 +51,23 @@ abstract class TokenCommand(open val token: IssuedTokenType, internal val inputI
...
 
-
 }
 
+/**
+ * Used when issuing [FungibleToken]s or [NonFungibleToken]s.
+ *
+ * @property token the group of [IssuedTokenType]s this command should be tied to.
+ * @property outputs the output state indices this command applies to.
+ * @param T the [TokenType].
```
Comment (kasiastreich) [inline contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt]: There is no T the [TokenType] anymore

File: contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt [hunk 2]
```diff
@@ -51,15 +51,23 @@ abstract class TokenCommand(open val token: IssuedTokenType, internal val inputI
...
+ */
 class IssueTokenCommand(override val token: IssuedTokenType, val outputs: List<Int> = listOf()) : TokenCommand(outputIndicies = outputs, token = token)
 
 /**
  * Used when moving [FungibleToken]s or [NonFungibleToken]s.
  *
  * @property token the group of [IssuedTokenType]s this command should be tied to.
+ * @property inputs the input state indices this command applies to.
+ * @property outputs the output state indices this command applies to.
  * @param T the [TokenType].
```
Comment (kasiastreich) [inline contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt]: There is no T the [TokenType] anymore

File: contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt [hunk 3]
```diff
@@ -68,6 +76,8 @@ class MoveTokenCommand(override val token: IssuedTokenType, val inputs: List<Int
  * Used when redeeming [FungibleToken]s or [NonFungibleToken]s.
  *
  * @property token the group of [IssuedTokenType]s this command should be tied to.
+ * @property inputs the input state indices this command applies to.
+ * @property outputs the output state indices this command applies to.
  * @param T the [TokenType].
```
Comment (kasiastreich) [inline contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt]: There is no T the [TokenType] anymore

File: contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt (hunk 1 above)
Comment (roger3cev) [reply contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/commands/TokenCommand.kt]: ah yes thanks

**Comment by roger3cev** [comment]
thanks for review - over to you @kasiastreich !

**Comment by roger3cev** [comment]
Thanks! :uni

**Pull request merged**